README.md

```diff
@@ -1,6 +1,5 @@
-[<img alt="libfabric master branch Travis CI status" src="https://travis-ci.org/ofiwg/libfabric.svg?branch=master"/>](https://travis-ci.org/ofiwg/libfabric)
 [<img alt="libfabric Coverity scan build status" src="https://scan.coverity.com/projects/4274/badge.svg"/>](https://scan.coverity.com/projects/4274)
```

Comment:
If you want a badge for the Github build check actions, you can use this:

```
[![Build Checks](https://github.com/ofiwg/libfabric/actions/workflows/pr-ci.yml/badge.svg)](https://github.com/ofiwg/libfabric/actions/workflows/pr-ci.yml)
```
Eg:
[![Build Checks](https://github.com/ofiwg/libfabric/actions/workflows/pr-ci.yml/badge.svg)](https://github.com/ofiwg/libfabric/actions/workflows/pr-ci.yml)

Reply:
Added via #7392 -- thanks!